src/cascadia/TerminalSettingsModel/ActionMap.cpp

```diff
@@ -683,18 +683,20 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
     // - keys: the key chord of the command to search for
     // Return Value:
     // - the command with the given key chord
-    // - nullptr if the key chord is explicitly unbound
+    // - nullptr if the key chord doesn't exist
     Model::Command ActionMap::GetActionByKeyChord(Control::KeyChord const& keys) const
     {
-        // Check the current layer
-        const auto cmd{ _GetActionByKeyChordInternal(keys) };
-        if (cmd.has_value())
+        const auto modifiers = keys.Modifiers();
+
+        if (auto vkey = keys.Vkey())

```

Comment:
would it be tremendously dumb to oneline this as 

```c++
        return _GetActionByKeyChordInternal({ modifiers, keys.Vkey(), 0 }).value_or( _GetActionByKeyChordInternal({ modifiers, 0, keys.ScanCode() }).value_or(nullptr));
```

Reply:
It would certainly be nice! But it would also eagerly check both variants, before returning either, and I'm not sure if that's a good idea...